### DISCUSSION TITLE
Supported Device list? Intel AMT/MEBX/vPRO.

### DISCUSSION THREAD

**Discussion by JSuenram** [Ylianst/MeshCentral#3627] [Q&A] [answered]
As it is partially hard to get device lists from several Vendors if a specific PC is fully AMT/MEBX capable maybe we can collect 
Vendor and Features supported here:

**Comment by jsastriawan** [comment, marked as answer]
MeshCmd and MeshAgent have AMT detection capabilities through RMCP network scanning and reading SMBIOS type 130 and type 131. Some ISV actually developed and published AMT scanning tool (boardgen/vtul) https://boardgent.com/network-scanner. I hope Ricardo Polo is watching this repo too.😀

See amtscanner.js in main folder, amt-scanner.js in agent module and smbios.js.

**Reply by JSuenram** [reply]
Yes, but to get the capabilities before you ever buy a PC it would be nice to now.... Hey, this Lenovo, Dell whatever XYZ Desktop has it or not...

**Reply by rapi3** [reply]
It is a good idea considering that some computers can have AMT but because of BUG's in BIOS/AMT can't be used at full: like Lenovo T410 with AMT 6.2.61 that have problems with CIRA

**Reply by jsastriawan** [reply]
Typically if your system has vPro logo, it should have full AMT support. To get that logo, OEM must pass verification test. While this is not 💯 percent fool proof at least you can ask OEM to fix the issue for you if there is a discrepancy.

For IOT system, it is slightly hard but typically we can check for Q-chipset and NIC with suffix LM in product spec/datasheet.

**Comment by Ylianst** [comment]
For Intel AMT, MeshCentral and MeshCommander should support AMT v5 and higher. However, I have a AMT v7 machine that I use for testing and it's the lowest version I test more regularity with. In any case, any decent Intel AMT machine from the last 5 years will likely do just fine. Note that there are Intel Standard Manageability (Intel SM) machines that are like AMT but without KVM, CIRA and other features. People sometimes will buy these and notice no "Remote Desktop" feature in MeshCommander. This is normal, these machine don't support this. So, make sure you get Intel AMT or Intel vPro to get KVM.
